app/actors/hyrax/actors/base_actor.rb

```diff
@@ -71,7 +71,8 @@ def apply_deposit_date(env)
       def save(env, use_valkyrie: false)
         return env.curation_concern.save unless use_valkyrie
 
-        resource = valkyrie_save(resource: env.curation_concern.valkyrie_resource)
+        # don't run validations again on the converted object if they've already passed
+        resource = valkyrie_save(resource: env.curation_concern.valkyrie_resource, valid: env.curation_concern.save)
```

Comment:
in the event `env.curation_concern` is an AF object, BUT `Hyrax.config.use_valkyrie?` is TRUE...we need the validation to run in `Wings::Valkyrie::Persister#save` because they were NOT run on the valkyrie object. otherwise, no errors are returned although they should be.

in this case, `env.curation_concern.save` will equal false.